library/Zend/View/Helper/HtmlTag.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2014 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
No, it will be done after merge. I don't feel like burdening existing PRs with these changes is needed at all ;-)